suite-native/module-assets/src/screens/AssetsImport.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
Suffix all screens with Screen otherwise it could be easily confused with components